### PR TITLE
baklava: fix cpu hogging

### PR DESCRIPTION
**Pull request by amitojsingh366**
- bump globkey
- potentially fix cpu hogging

### CODE REVIEW

File: baklava/src/electron.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ function createMainWindow() {
     mainWindow.webContents.openDevTools();
   }
   mainWindow.loadURL(
-    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
+    !__prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
```
Comment (theotterlord) [inline baklava/src/electron.ts:91]:
```suggestion
    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000"
```

**Review by TheOtterlord — CHANGES_REQUESTED** [review]

File: baklava/src/electron.ts (hunk 1 above)
Comment (amitojsingh366) [reply baklava/src/electron.ts:91]: yea i noticed this, anyway this is still a draft cuz i had to sleep and do a few more tests

**Comment by benawad** [comment]
@amitojsingh366 did you make this a draft on purpose?

**Comment by amitojsingh366** [comment]
@benawad yes, @StratusFearMe21 is working on `globkey` the rust based keybind listener we are using *(he's adding optimizations)*, he needs some time so this is draft till then.

**Comment by amitojsingh366** [comment]
closing this for now since i've gotten no response from @StratusFearMe21

**Pull request closed without merging**